### ISSUE TITLE
Cannot delete images from post, "delete" icon hidden under "alt text" icon

### ISSUE THREAD

**Issue by RainbowPunkStar** [glitch-soc/mastodon#2982]
### Steps to reproduce the problem

1. Added 4 images to a post
2. I had a duplicate so I wanted to delete it to add another one
3. Impossible to click on the "delete" icon as it stand under the "alt text" icon in the image thumbnail
4. Tried to delete text to make the thumbnails bigger but even with empty post, the "delete" icon is still inaccessible.

### Expected behaviour

Image should be big enough so the icons do not cross each other and "delete" icon should always be accessible.

### Actual behaviour

The thumbnail is too small so the "delete" icon is going under the "alt text" icon and I cannot click it.

### Detailed description

When I add 4 images to a post on web interface, I cannot access the delete icon, which lays under the alt text icon in the thumbnail. It's even worse when I add text to the post, making the image further smaller and then all the 3 icons (delete, alt text, modify) merge together. Please make the minimum size of images thumbnails bigger so the icons do not cross each other.

![Image](https://github.com/user-attachments/assets/abcd3528-41d7-450a-bce5-e518339afdd3)

![Image](https://github.com/user-attachments/assets/dd990690-901c-4558-8e26-89305be35428)

### Mastodon instance

eldritch.cafe

### Mastodon version

v4.3.4+glitch

### Browser name and version

Firefox 135.0.1 (64 bits)

### Operating system

Windows 10

**Comment by ClearlyClaire** [comment]
Oops, this has been fixed upstream but I forgot to port the patch to glitch-soc. I have done so today, and it should be fixed in the next release.
